python/interpret_community/__init__.py

```diff
@@ -18,5 +18,8 @@
 # Only log to disk if environment variable specified
 interpretc_logs = os.environ.get('INTERPRETC_LOGS')
 if interpretc_logs is not None:
-    logging.basicConfig(filename=interpretc_logs, level=logging.INFO)
-    logging.info('Initializing logging file for interpret-community')
+    logger = logging.getLogger(__name__)
+    handler = logging.FileHandler(interpretc_logs)
+    handler.setLevel(logging.INFO)
+    logger.addHandler(handler)
+    logger.info('Initializing logging file for interpret-community')
```

Comment:
lgtm
For a future pr, I suggest prepending the first (at least the first) logging message with [name]: so it's easier to search for in the log file, it '[interpret-community]: Initializing logging file for interpret-community'